### DISCUSSION TITLE
전역데이터 새로고침시

### DISCUSSION THREAD

**Discussion by Hyevvy** [prgrms-web-devcourse/Team-BackFro-ArtZip-FE#148] [03.Technical]
- 새로고침할 때마다 token을 통해서 userId를 다시 받아온다.
   - 로컬스토리지에 accessToken을 담는 것으로 똑같이!   

- 만료시간 체크하는 부분에 있어서는 axios request interceptor를 통해서 프론트단에서 만료되었는지 한 번 검증 후 만료되었다면 accessToken 과 responseToken을 보내어 새로 accessToken을 발급받는다. 
-> 이 부분에 대해서 api 추가가 필요함

**Comment by punchdrunkard** [comment]
흑흑 next 라서  ssr에서 로컬스토리지를 못 읽는 이슈가 있어서 ... 쿠키로 구현하게 되었습니다...
